markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+---
+title: Step-by-step guide to writing an institutional profile
+subtitle: Walkthrough on what you need to set up an nf-core institutional profile
+---
+
+## Introduction
+
+nf-core offers a centralised place to store Nextflow configuration profiles that work at an _institutional_ level.
+
+What this means is that you can specify common Nextflow pipeline configurations and options that can be shared across all users of your particular institutional cluster.
+
+nf-core offers two level of profile sharing: institutional and pipeline-specific profiles via [nf-core/configs](https://github.com/nf-core/configs)
```

Comment:
I would rephrase this part, I would not categorise into institutional and pipeline-specific, but global and pipeline-specific. 

```suggestion
nf-core offers two level of institutional profile sharing: global and pipeline-specific profiles via [nf-core/configs](https://github.com/nf-core/configs)
```

Pipeline-specific are still institutional, (except for the viralrecon genomes.config file that is global but pipeline specific.

Reply:
But I recognise the fact that we actually use institutional most of the time to mean what I called global

Reply:
What about: `global institutional config` vs . `pipeline-specific institutional config` throughout the text?

Reply:
Sounds good to me, maybe we should update the nf-core/configs repo for that too?

Reply:
Probably should! Will do after this is in